artichoke-backend/build.rs

```diff
@@ -0,0 +1,117 @@
+#![deny(clippy::all, clippy::pedantic)]
+#![deny(warnings, intra_doc_link_resolution_failure)]
+#![doc(deny(warnings))]
+
+use fs_extra::dir;
+// use fs_extra::file::{self, CopyOptions};
+use std::env;
+use std::fs;
+use std::path::Path;
+use std::process::Command;
+
+/// Path helpers
+struct Build;
+
+impl Build {
+    fn root() -> String {
+        env::var("CARGO_MANIFEST_DIR").unwrap()
+    }
+
+    fn generated_dir() -> String {
+        format!("{}/src/generated", env::var("OUT_DIR").unwrap())
+    }
+
+    fn ruby_source_dir() -> String {
+        format!("{}/ruby_2_6_3", env::var("OUT_DIR").unwrap())
+    }
+
+    fn ruby_vendored_dir() -> String {
+        format!("{}/vendor/ruby_2_6_3", &Build::root())
+    }
+
+    fn patch(patch: &str) -> String {
+        format!("{}/vendor/{}", Build::root(), patch)
+    }
+
+    fn get_package_files(package: &str) -> String {
+        let output = Command::new("bash")
```

Comment:
can we invoke `ruby` directly instead of indirecting through `bash`?

This might be a cargo cult from the patch logic, which includes the bash bits to more easily implement stdin redirection.